app/models/cuisine.rb

```diff
@@ -0,0 +1,15 @@
+# == Schema Information
+#
+# Table name: cuisines
+#
+#  id            :integer          not null, primary key
+#  name          :string
+#  icon          :string
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
+#  restaurant_id :integer
+#
+
+class Cuisine < ApplicationRecord
+  belongs_to :restaurant
```

Comment:
Are you sure this is the right relation?
see https://guides.rubyonrails.org/association_basics.html#the-belongs-to-association

Reply:
You are correct. fixed in the new commit